servers_v6.json

```diff
@@ -54,5 +54,9 @@
   {
      "name": "MindustryBR",
      "address": ["mindustryptbr.ddns.net", "mindustryptbr.ddns.net:4444", "mindustryptbr.myddns.me:6666", "mindustryptbr.myddns.me:5555"]
+  },
+  {
+     "name": "ALEX",
+     "address": ["209.127.178.13:6568", "209.127.178.13:6569"]
```

Comment:
> `209.127.178.13`

are you sure you want to hardcode the ips instead of using a dns record?

Reply:
yes for now. ill update with a www once its debugged.

Reply:
i tried using dns, but the dns dont work... any suggestions? sorry im noob with dns stuff ;-;

![image](https://user-images.githubusercontent.com/67626131/101664549-f11a6680-3a86-11eb-9110-3b9e4f09b768.png)

with this set up, i typed pvp.alexmindustry.freeddns.org into the "Add server" in v6 but it doesnt show up
any help / suggestion is appreciated

Reply:
i tried with ddns.net as well, but it dont work. is it because of the "TYPE" ?
![image](https://user-images.githubusercontent.com/67626131/101853062-f7e5ce00-3b99-11eb-9f3c-123a520d1a32.png)